test/demo/utils/fdc3RemoteExecution.ts

```diff
@@ -270,6 +270,17 @@ export async function findIntentsByContext(executionTarget: Identity, context: C
     }, context).catch(handlePuppeteerError);
 }
 
+export async function clickHTMLElement(executionTarget: Identity, elementSelector: string): Promise<boolean> {
```

Comment:
Puppeteer has some fairly good builtins for DOM interaction which we're using to do things similar to this in notifications testing, but probably not worth the effort of changing everything now that this is written.